protocol/protobuf/communities.proto

```diff
@@ -278,3 +278,7 @@ message CommunityUpdateGrant {
   bytes community_id = 2;
   map<uint32, bytes> grants = 3;
 }
+
+message RequestCommunityEncryptionKeys {
```

Comment:
it's a matter of taste, usually we place request in the end: `CommunityEncryptionKeysRequest`